### ISSUE TITLE
Multi-word Command Feature

### ISSUE THREAD

**Issue by dogukanarat** [funbiscuit/embedded-cli#52]
Hi,

As latest version of the repository can accept only one word command. Is there any way to support multi-word command feature.

Example usage:
`uart send <ifc> <data>`

**Comment by funbiscuit** [comment]
No, that's not possible with current implementation (PR's are welcome).
For now I can only suggest using dashes: `uart-send <ifc> <data>`
